src/OpenTelemetry.Instrumentation.AspNet/Implementation/HttpInListener.cs

```diff
@@ -97,7 +97,7 @@ public override void OnStartActivity(Activity activity, object payload)
             activity.DisplayName = path;
 
             activity.SetKind(ActivityKind.Server);
-
+            activity.SetCustomProperty("OTel.HttpInHandler.Request", request);

```

Comment:
Do we need this change? I don't think we added this for the other example Amrit worked on?

Reply:
this is an accident